ansible/README.rst

```diff
@@ -27,25 +27,32 @@ Host machine requirements
 
 The recommended emulation target requirements:
 
-- 1 deploy node to run ansible and Rally workload
+- 1 deploy node to run ansible deployment script
 - 1 OVN database node
-- 2 OVN chassis host to run emulated OVN chassis container
+- 2 hosts to run emulated OVN chassis container
 - 1 host to run rally container (could be the same as the OVN database node)
 
 .. image:: ovn-emulation-deployment.png
    :alt: OVN emulation deployment
 
-  
+
+.. note:: The above setting is just an example. For example, all OVN emulated chassis could be run on one host. Further, an all-in-one configuration deploys all containers on a single host.
```

Comment:
Thanks for reviewing the PR, @mestery !

When the line is wrapped at 80 chars, the file could not be rendered correctly as an online rst file, especially for some rst symbols e.g., the above .. note::. Do you think we need to remove all rst symbols in this README file?